1-JavaScript/Week4/homework/ex3-lemonAllergy.test.js

```diff
@@ -26,26 +26,29 @@ const fruitBasket = [
 ];
 
 // ! Function under test
-function sanitizeFruitBasket(/* TODO parameter(s) go here */) {
-  // TODO complete this function
+function sanitizeFruitBasket(fruitBasket, fruit) {
+ const filteredFruit= fruitBasket.filter(item => item !== fruit);
+ return filteredFruit;
```

Comment:
no need to create a const here since we're not re-using the value within this closure. Thus 
```suggestion
 return fruitBasket.filter(item => item !== fruit);
```

Reply:
i tried to express this code line meaning filtered fruit. But i changed them what you said and i added comment line  instead of create new const variable👍